src/pages/Transactions/IssueRequestsTable/IssueRequestModal/index.tsx

```diff
@@ -1,194 +1,30 @@
 import * as React from 'react';
-import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 import clsx from 'clsx';
-import { IssueStatus } from '@interlay/interbtc-api';
 
-import BTCPaymentPendingStatusUI from './BTCPaymentPendingStatusUI';
-import IssueRequestStatusUI from './IssueRequestStatusUI';
-import WhoopsStatusUI from './WhoopsStatusUI';
 import RequestModalTitle from '../../RequestModalTitle';
-import PriceInfo from 'pages/Bridge/PriceInfo';
+import IssueUI from 'components/IssueUI';
 import CloseIconButton from 'components/buttons/CloseIconButton';
 import Hr1 from 'components/hrs/Hr1';
-import Hr2 from 'components/hrs/Hr2';
 import InterlayModal, { Props as ModalProps, InterlayModalInnerWrapper } from 'components/UI/InterlayModal';
-import { WrappedTokenAmount, WRAPPED_TOKEN_SYMBOL } from 'config/relay-chains';
-import { POLKADOT, KUSAMA } from 'utils/constants/relay-chain-names';
-import { displayMonetaryAmount, getUsdAmount, shortAddress } from 'common/utils/utils';
-import { StoreType } from 'common/types/util.types';
-import { ReactComponent as BitcoinLogoIcon } from 'assets/img/bitcoin-logo.svg';
-import { getColorShade } from 'utils/helpers/colors';
-
-// TODO: should type properly (`Relay`)
-const renderModalStatusPanel = (request: any) => {
-  switch (request.status) {
-    case IssueStatus.PendingWithBtcTxNotFound: {
-      return <BTCPaymentPendingStatusUI request={request} />;
-    }
-    case IssueStatus.RequestedRefund: {
-      return <WhoopsStatusUI request={request} />;
-    }
-    default: {
-      return <IssueRequestStatusUI request={request} />;
-    }
-  }
-};
 
 interface CustomProps {
```

Comment:
Could be renamed to `IssueRequestModalProps`

Reply:
Actually, it was there before. I have not touched it. `CustomProps` is not for the whole `IssueRequestModal`.
`CustomProps & Omit<ModalProps, 'children'>` could be `IssueRequestModalProps` for the whole component. Besides that, according to the convention, we name types of components props just `Props` in order to avoid maintaining that type name together with the component name.